### PR TITLE
[portBLAS]Rename SYCL-BLAS to portBLAS

### PR DESCRIPTION
**Pull request by aacostadiaz**
This PR changes the name of SYCL-BLAS to portBLAS.

- [x] Do all unit tests pass locally? Attach a log. ([Test Results.html.zip](https://github.com/oneapi-src/oneMKL/files/12370844/Test.Results.-.All_CTest_CPU.html.zip))
- [x] Have you formatted the code using clang-format?

### CODE REVIEW

File: src/blas/backends/portblas/CMakeLists.txt [hunk 1]
```diff
@@ -62,108 +62,108 @@ if(SYCLBLAS_TUNING_TARGET)
...
-  # Set the policy to forward variables to SYCL-BLAS configure step
+  # Set the policy to forward variables to portBLAS configure step
   set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
   set(FETCHCONTENT_BASE_DIR "${CMAKE_BINARY_DIR}/deps")
   FetchContent_Declare(
-    sycl-blas
+    portBLAS
     GIT_REPOSITORY https://github.com/codeplaysoftware/sycl-blas
-    GIT_TAG        master
+    GIT_TAG        0c38d7db14f82e4c66047237e321d793f5316439
```
Comment (aacostadiaz) [inline src/blas/backends/portblas/CMakeLists.txt]: I'll update this with the right repository and tag when https://github.com/codeplaysoftware/sycl-blas/pull/455 is merged.

Comment (aacostadiaz) [reply src/blas/backends/portblas/CMakeLists.txt]: Done

**Review by muhammad-tanvir-1211 — APPROVED** [review]
LGTM, Thanks.

**Review by andrewtbarker — APPROVED** [review]
This looks fine, thanks!

**Review by mkrainiuk — APPROVED** [review]

**Pull request merged**